demo/project_demo05-DebeziumMySQL/run.py

```diff
@@ -0,0 +1,193 @@
+import os
+import sys
+import subprocess
+import time
+import json
+
+from dbsp import DBSPPipelineConfig
+from dbsp import JsonInputFormatConfig, JsonOutputFormatConfig
+from dbsp import KafkaInputConfig
+from dbsp import KafkaOutputConfig
+
+sys.path.append(os.path.join(os.path.dirname(os.path.dirname(__file__)), ".."))
+from demo import *
+
+SCRIPT_DIR = os.path.join(os.path.dirname(__file__))
+
+
+def prepare(args=[]):
+    connect_server = os.getenv("KAFKA_CONNECT_SERVER", "http://localhost:8083")
+
+    print("Delete old connector")
+    # Delete previous connector instance if any.
+    # Note: this won't delete any existing Kafka topics created
+    # by the connector.
+    subprocess.run(
+        f"curl --request DELETE --url {connect_server}/connectors/inventory-connector",
+        shell=True,
+        check=False,
+    )
+
+    print("Create connector")
+    # Create connector.  The new connector will continue working with
+    # existing Kafka topics created by the previous connectors instance
+    # if the exist.
+    cmd = (
+        f"""curl --fail-with-body --request POST --url {connect_server}"""
```

Comment:
Seems strange to run curl via a subprocess and then parse the output stream to see if the call worked. Why not use https://docs.python-requests.org/en/latest/index.html?

Reply:
good point, don't know what I was thinking